spring-cloud-dataflow-dependencies/pom.xml

```diff
@@ -20,7 +20,7 @@
 		<spring-batch-admin-dependencies.version>1.3.1.RELEASE</spring-batch-admin-dependencies.version>
 		<spring-cloud-dataflow-ui.version>1.0.0.RELEASE</spring-cloud-dataflow-ui.version>
 		<spring-shell.version>1.2.0.RELEASE</spring-shell.version>
-		<spring-cloud-deployer.version>1.0.1.RELEASE</spring-cloud-deployer.version>
+		<spring-cloud-deployer.version>1.0.2.BUILD-SNAPSHOT</spring-cloud-deployer.version>
```

Comment:
TBD: this will use spring-cloud-deployer `1.0.2.RELEASE` once that is available.